OutOfSchool/OutOfSchool.DataAccess/Models/Provider.cs

```diff
@@ -9,13 +9,13 @@ public class Provider
     {
         public long Id { get; set; }
 
-        [Required(ErrorMessage = "Title is required")]
+        [Required(ErrorMessage = "Full Title is required")]
         [DataType(DataType.Text)]
         [MaxLength(60)]
         [MinLength(1)]
-        public string Title { get; set; }
+        public string FullTitle { get; set; }
 
-        [Required(ErrorMessage = "ShortTitle is required")]
+        [Required(ErrorMessage = "Short Title is required")]
```

Comment:
do we have localization support in the project yet?

Reply:
Yes we have, but not to all parts of the project.